ts/Data/README.md

```diff
@@ -211,8 +211,8 @@ either an URL or a local source.
 
 ### DataConnector Registry
 
-DataConnector types can be directly loaded via import. In case of bundles
-connectors can also accessed via registry, as the registry gets updated with
+DataConnector types can be directly loaded via import. In case of bundles,
```

Comment:
There should be no comma. Instead continue with ...the connectors...

Reply:
I think there should be a comma here, it's more clear. Also, perhaps add a _the_ to the introductory phrase ("In the case of bundles").